libc/printf/stdio_printf.c

```diff
@@ -211,7 +211,7 @@ static char *test_unsignedToStr(unsigned long long int value, int base, bool big
 
 static char *test_intToAscii(int value)
 {
-	char *str = malloc(2 * sizeof(char));
+	static char str[2];
 	str[0] = value;
 	str[1] = '\0';
 	return str;
```

Comment:
Unfornutately as it's mentioned in our coding convention: 
![Screenshot from 2023-07-17 17-59-44](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/77304431/9ce12321-5f39-41f7-b66f-92c22a20e78d)



https://github.com/phoenix-rtos/phoenix-rtos-doc/blob/master/coding.md